msdk-id-sirius/pom.xml

```diff
@@ -9,40 +9,105 @@
 
 
   <repositories>
-    <!--<repository>-->
-      <!--<id>bioinf-jena</id>-->
-      <!--<name>bioinf-jena-release</name>-->
-      <!--<url>http://bio.informatik.uni-jena.de/repository/libs-release-local</url>-->
-    <!--</repository>-->
-
     <repository>
-      <id>bioinf-jena-sirirusCli</id>
+      <id>bioinf-jena-main</id>
+      <name>bioinf-jena-release</name>
+      <url>https://bio.informatik.uni-jena.de/repository/libs-release-local</url>
+    </repository>
+    <repository>
+      <id>bioinf-jena</id>
       <name>bioinf-jena-release</name>
-      <url>http://bio.informatik.uni-jena.de/artifactory/libs-releases-local</url>
+      <url>https://bio.informatik.uni-jena.de/repository/local</url>
```

Comment:
Do you need this local repository? Shouldn't all the relevant artifacts be in the libs-release-local repo?

Reply:
The three repos required to build were taken from: https://github.com/boecker-lab/sirius/blob/release/build.gradle

Reply:
Relevant portion:
```groovy
repositories {
        mavenLocal()
        mavenCentral()
        maven {
            url "https://bio.informatik.uni-jena.de/repository/local"
        }
        maven {
            url "http://www.ebi.ac.uk/Tools/maven/repos/content/groups/ebi-repo/"
            //url "http://www.ebi.ac.uk/intact/maven/nexus/content/repositories/ebi-repo"
        }
        maven {
            url "https://www.xypron.de/repository" //glpk repo
        }
    }
```